### ISSUE TITLE
Check if the repo has a changelog and include it in a link somehow?

### ISSUE THREAD

**Issue by orta** [ashfurrow/twelease#8]
Could be an ENV var to enable, and then postfixed to the templated message?

**Comment by ashfurrow** [comment]
Makes sense, there's probably no harm in checking anyway, and then supplying a URL to the Moustache template, right?
